Sources/PeerConnectionManager+Peer.swift

```diff
@@ -90,7 +90,28 @@ public extension PeerConnectionManager {
     // - parameter toPeers: Specified `Peer` objects to send data.
 
     func sendData(_ data: Data, toPeers peers: [Peer] = []) {
-        session.sendData(data, toPeers: peers)
+        var peerRequested = peers.isEmpty == true ? allAvailablePeers : peers
+        var sessions: [PeerSession] = allAvailableSessions.reduce([]) { (sessions, serviceSession) in
+            if serviceSession.connectedPeers.first(where: { return peerRequested.contains($0) }) != nil {
+                return sessions + [serviceSession]
+            }
+
+            if peerRequested.contains(serviceSession.servicePeer) {
+                return sessions + [serviceSession]
+            }
+
+            return sessions
+        }
+
+        for session in sessions {
+            let peerInSession = session.connectedPeers.filter { peerRequested.contains($0) }
+            peerRequested = Array(Set(peerRequested).subtracting(peerInSession))
+
+            guard peerInSession.count > 0 else { continue }
```

Comment:
`isEmpty == false`

Reply:
yea, I though of using `isNotEmpty` but it's a `TillerFoundation` feature which I do not have access here.

Forgot that I could use `isEmpty`